Sources/Kumo/Blobs/Storage/InMemory.swift

```diff
@@ -20,31 +18,35 @@ class InMemory: StorageLocation {
     private let backingCache = NSCache<NSString, Reference>()
 
     private var keys = Set<String>()
+    private var queue = DispatchQueue(label: "key_queue")
     weak var delegate: StoragePruningDelegate?
 
-    func fetch<D: _DataRepresentable>(for url: URL, arguments: D._RepresentationArguments) throws -> D? {
+    func fetch<D: _DataRepresentable>(for url: URL, arguments _: D._RepresentationArguments) throws -> D? {
```

Comment:
should this be `_ arguments`?

Reply:
Oh, it's because arguments aren't used, eh

Reply:
Yeah, I've taken to using swiftformat when I'm tired of fighting Xcode's auto-formatting attempts and it does this when arguments are unused. I think I like it but we don't do it elsewhere generally in function definitions.